### DISCUSSION TITLE
I want to delete the organization created under a user account

### DISCUSSION THREAD

**Discussion by veriprofile** [community/community#72948] [New to GitHub]
I don't see any delete option or danger zone in settings of an organization, how do I delete it?

**Comment by vlastami** [comment]
1.   Log in to your GitHub account associated with that organization.
2.   Navigate to the organization's page by clicking on its name in the top-right corner of the screen.
3.   Click on the Settings tab for the organization. This should be on the right-hand side of the screen.
4.   Scroll down on the settings page to the Danger Zone section at the bottom. **(If you're not seeing the Danger Zone, make sure you have the appropriate permissions for the organization. Only owners have the ability to delete an organization.)**
5.   In the Danger Zone, you'll see an option to Delete this organization. Click on it.
6.   GitHub will ask you for confirmation and might require you to enter your password and/or perform additional verification steps.
